client/src/components/Edit.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { useEffect, useState } from 'react'
+
+
+export default function Edit(props) {
+    //<----- Dig into passed info -----//
+    let info = props.postObj
+
+    //<----- Toggle Delete Button ----->//
+    const [delButt, setDelButt] = useState(false)
+    function toggleDel(evt) {
```

Comment:
This function would still work without the `evt` on line 11. The event is not used in this function. onClick on line 21 is handling the "event" in this case and useState is handling the rest through the `setDelButton` function.